src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java

```diff
@@ -142,7 +143,7 @@ public void setSize(int size) {
         }
 
         this.size = size;
-        cfg.setValue("backpacks." + id + ".size", size);
+        cfg.setValue(PlayerBackpack.backpacks + id + ".size", size);
```

Comment:
`PlayerBackpack` is redundant here, no need to specify the class you are already in.